cucumber/authenticators_jwt/features/authn_jwt_fetch_signing_key.feature

```diff
@@ -0,0 +1,535 @@
+Feature: JWT Authenticator - Fetch signing key
+
```

Comment:
Cant we add background with all steps and policy part that is commont for all the scenarios?

Reply:
every scenario has a different policy so i dont think that would be a good idea, also the dynamic tests policies are complicated enough even without the background